openstf/model/model_creator.py

```diff
@@ -102,18 +103,22 @@ def create_model(model_type: Union[MLModelType, str], **kwargs) -> OpenstfRegres
             # This will raise a ValueError when an invalid model_type str is used
             # and nothing when a MLModelType enum is used.
             model_type = MLModelType(model_type)
+            model_class = ModelCreator.MODEL_CONSTRUCTORS[model_type]
+            valid_kwargs = valid_model_kwargs[model_type]
         except ValueError as e:
-            valid_types = [t.value for t in MLModelType]
-            raise NotImplementedError(
-                f"No constructor for '{model_type}', "
-                f"valid model_types are: {valid_types}"
-            ) from e
+            if is_custom_type(model_type):
```

Comment:
This custom type will become standard behavior. So it's nicer if we can load the custom model in the try statement instead of in except statement. Then the except statement has the logic for error handling